docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,505 @@
+      },
+      {
+        name: 'href',
+        type: 'string',
+        description: `When supplied, wraps the component in a link, and directs users to the url when item is selected. See the [role](#Role) variant to learn more.`,
+      },
+      {
+        name: 'addCollaborators',
+        type: 'boolean',
+        description: `When supplied, it appends an 'add' icon to the avatar pile as a call to action to the user. See [Best Practices] (#Best-practices) for more info.`,
+      },
+      {
+        name: 'ref',
+        type: `React.Ref<'div'> | React.Ref<'a'>`,
+        description: `Forward the ref to the underlying div or anchor element. See the [role](#Role) variant to learn more.`,
+        href: 'ref',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+AvatarGroup requires \`accessibilityLabel\`. AvatarGroup is a group of elements that require a parent label describing both the data presented and the call to action in the case of button and link roles. As seen in the example below, the screen-reader reads: "Group collaborators: Keerthi, Alberto, and another 10 collaborators. Add collaborators to this board."
+
+
+If AvatarGroup is used as a control button to show/hide Popover-component, we recommend passing the following ARIA attributes to assist screen readers:
+
+- \`accessibilityControls\`: informs the screen reader that AvatarGroup controls the display of an anchored Popover-component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-component attached to AvatarGroup. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const [selectedBoard, setSelectedBoard] = React.useState('Fashion');
+  const anchorRef = React.useRef();
+
+  const SearchCollaboratorsField = () => {
+    const ref = React.useRef();
+
+    React.useEffect(() => {
+      ref.current.focus();
+    }, []);
+
+    return (
+      <SearchField
+        accessibilityLabel="Search other users"
+        id="searchField"
+        onChange={() => {}}
+        placeholder="Search by name or email"
+        size="lg"
+        ref={ref}
+      />
+    )
+  }
+
+  return (
+    <React.Fragment>
+      <Box>
+        <AvatarGroup
+          accessibilityLabel="Group collaborators: Keerthi, Alberto, and another 10 collaborators. Add collaborators to this board."
```

Comment:
Copying here from our Slack convo: reminder to update the labels in these examples to something more like "Collaborators: Keerthi, Alberto, and 10 more. Add collaborators to this board."

Reply:
DONE

Reply:
![image](https://user-images.githubusercontent.com/10593890/116589759-f91a5c00-a8ea-11eb-81f7-39145ffebf52.png)

Reply:
I would still get rid of the "Group" too, Collaborators already implies multiple in this case